tests/integration/test_execd/test_run_active_response/test_firewall_drop.py

```diff
@@ -0,0 +1,142 @@
+'''
+copyright: Copyright (C) 2015-2023, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: Active responses execute a script in response to the triggering of specific alerts based
+       on the alert level or rule group. These tests will check if the 'active responses',
+       which are executed by the 'wazuh-execd' daemon via scripts, run correctly.
+
+components:
+    - execd
+
+suite: execd
+
+targets:
+    - agent
+
+daemons:
+    - wazuh-analysisd
+    - wazuh-execd
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - Debian Buster
+    - Red Hat 8
+    - Ubuntu Focal
+    - Ubuntu Bionic
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/capabilities/active-response/#active-response
+'''
+import pytest
+
+from pathlib import Path
+
+from wazuh_testing.constants.paths.logs import WAZUH_LOG_PATH, ACTIVE_RESPONSE_LOG_PATH
+from wazuh_testing.modules.active_response import patterns as ar_patterns
+from wazuh_testing.modules.execd import patterns as execd_paterns
+from wazuh_testing.modules.execd import EXECD_DEBUG_CONFIG
+from wazuh_testing.tools.file_monitor import FileMonitor
+from wazuh_testing.utils.callbacks import generate_callback
+from wazuh_testing.utils.configuration import get_test_cases_data, load_configuration_template
+
+from . import CONFIGS_PATH, TEST_CASES_PATH
+
+
+# Set pytest marks.
+pytestmark = [pytest.mark.agent, pytest.mark.tier(level=1)]
+
+# Cases metadata and its ids.
+cases_path = Path(TEST_CASES_PATH, 'cases_execd_firewall_drop.yaml')
+config_path = Path(CONFIGS_PATH, 'config_run_active_response.yaml')
+test_configuration, test_metadata, cases_ids = get_test_cases_data(cases_path)
+test_configuration = load_configuration_template(config_path, test_configuration, test_metadata)
+
+# Test internal options and configurations.
+local_internal_options = EXECD_DEBUG_CONFIG
```

Comment:
Remember to change this to create the dict here.

Reply:
Applied on commit 51071c4c7601f7a486736ec2ad72f62d52673652